### PR TITLE
Valeria/hooks

### PR DESCRIPTION
**Pull request by svaleria**
- Current item added in config  and appconfig.json
- ItemPayload Added in views/models
- Recurrence Object added in views/models 
- create item before scenario hook implemented in Hooks

### CODE REVIEW

File: 2.Views/WebAppPages/HomePage.cs [hunk 1]
```diff
@@ -14,8 +15,35 @@ public class HomePage
...
+    public void SelectProject()
+    {
+        ProjectButton.Click();
+    }
+
+    public void AddItem(string itemName)
+    {
+        AddToDoInput.Type(itemName);
+        AddToDoInput.Type(Keys.Enter);
+    }
```
Comment (gioacchinogioffre) [inline 2.Views/WebAppPages/HomePage.cs:48]: These two methods should be in new classes on the Views/Actions/Items and Views/Actions/Project folders , at least that's what I think we defined for the methods 🧐

**Review by gioacchinogioffre — CHANGES_REQUESTED** [review]
Just one thing about the home methods that I commented. Another thing, did you merge your branch with main successfully ? Because the changes of the lasts merged PR's (base hooks and waitstrategy) shouldn't be here. Please try to review that and also check the conflicts. Thanks Vale!

File: 3.Tests/API/Features/Project/Post.feature [hunk 2]
```diff
@@ -2,30 +2,11 @@ Feature: Project Creation
 
     @acceptance
     Scenario: Create a new project with valid parameters and valid authentication
-        Given the user is authenticated
+        Given the user is authenticated with "Valeria.Gonzales@jala.university" and "1234"
```
Comment (sebasonti-jalauni) [inline 3.Tests/API/Features/Project/Post.feature:5]: Don't submit credentials on the code!

**Review by sebasonti-jalauni — CHANGES_REQUESTED** [review]
Good jobs, a ton of progress, just beware of committing credentials

File: 2.Views/WebAppPages/HomePage.cs (hunk 1 above)
Comment (svaleria) [reply 2.Views/WebAppPages/HomePage.cs]: solved

File: 2.Views/Models/ItemPayload.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
+﻿namespace Todoly.Views.Models;
+using System.Collections.Generic;
+public record ItemPayload
+{
```
Comment (gioacchinogioffre) [inline 2.Views/Models/ItemPayload.cs:4]: Why do you have a ItemPayload class? You could use the ItemModel of that folder

File: 3.Tests/UI/Features/User/Login.feature [hunk 4]
```diff
@@ -1,9 +1,9 @@
-Feature: User Login
-As a non-logged in user, the user should be able to login into the site
+# Feature: User Login
```
Comment (gioacchinogioffre) [inline 3.Tests/UI/Features/User/Login.feature:1]: Why are you commenting this feature? This has a test working already

File: 3.Tests/UI/Features/Settings/Profile/UpdateFullName.feature [hunk 5]
```diff
@@ -1,20 +1,20 @@
-Feature: Full name update
-    As a logged in user, the user should be able to update his full name from his profile settings.
+# Feature: Full name update
+#     As a logged in user, the user should be able to update his full name from his profile settings.
```
Comment (gioacchinogioffre) [inline 3.Tests/UI/Features/Settings/Profile/UpdateFullName.feature:2]: Why are you commenting this feature? This has a test working already

**Pull request closed without merging**